lib/stub_ui/lib/src/ui/text.dart

```diff
@@ -1559,6 +1615,57 @@ class ParagraphBuilder {
     _ops.add(text);
   }
 
+  /// Adds an inline placeholder space to the paragraph.
+  ///
+  /// The paragraph will contain a rectangular space with no text of the dimensions
+  /// specified.
+  ///
+  /// The [width] and [height] parameters specify the size of the placeholder rectangle.
```

Comment:
width, height, alignment, etc. are arguments to the function and not properties on the object. THey should be enclosed in ` instead of []